VIMVideoPlayer/VIMVideoPlayer.m

```diff
@@ -424,6 +426,8 @@ - (void)resetPlayerItemIfNecessary
 
 - (void)preparePlayerItem:(AVPlayerItem *)playerItem
 {
+    self.item = playerItem;
```

Comment:
In other words, you recommend overriding the `item` property's getter and adding / removing observers there? I was thinking about that, but not sure if that's preferable or not. Thoughts?